data-serving/data-service/schemas/cases.indexes.json

```diff
@@ -0,0 +1,45 @@
+[
+    {
+        "name": "textIdx",
+        "key": {
+            "notes": "text",
+            "revisionMetadata.creationMetadata.curator": "text",
+            "demographics.occupation": "text",
+            "demographics.nationalities": "text",
+            "demographics.ethnicity": "text",
+            "location.country": "text",
+            "location.administrativeAreaLevel1": "text",
+            "location.administrativeAreaLevel2": "text",
+            "location.administrativeAreaLevel3": "text",
+            "location.place": "text",
+            "location.name": "text",
+            "pathogen.name": "text",
+            "caseReference.sourceUrl": "text"
+        }
+    },
+    {
+        "name": "casesCaseRefIdx",
+        "key": {
+            "caseReference.sourceId": -1,
+            "caseReference.sourceEntryId": -1
+        },
+        "unique": true,
+        "partialFilterExpression": {
```

Comment:
Great side-benefit is that these opts are now in the same place as the key definition.

Reply:
:D